HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs

```diff
@@ -129,6 +130,23 @@ private void Edit_Display_Information(object sender, RoutedEventArgs e)
             EditMode = !EditMode;
         }
 
+        public void ChangeEditButtonVisibility() {
+            if (_selectedMapObject == null)
+            {
+                EditObjectButton.Visibility = Visibility.Hidden;
+            }
+            else
+            {
+                if (_selectedMapObject.MapObjectEntity.MapObjectType.TypeOfMapObject != TypeOfMapObject.ROAD)
+                {
+                    EditObjectButton.Visibility = Visibility.Visible;
+                }
+                else {
+                    EditObjectButton.Visibility = Visibility.Hidden;
+                }
+            }
+        }
+
         private void Change_Display_Information(object sender, RoutedEventArgs e)
```

Comment:
### Comment

This method is hard to read.

### Proposal

Don't write comments which will describe what this code doing, make little private methods which will describe it in her name, then this code will become more readable.

```sh
private void Change_Display_Information(object sender, RoutedEventArgs e){

  NewDescriptivePrivateMethod(..);
  AnotherDescriptivePrivateMethod(..); 
  NewAnotherDescriptivePrivateMethod(..);

}
```

Reply:
edited

Reply:
Great job @kmilena98 !